Makefile

```diff
@@ -26,6 +26,21 @@ build:
 install:
 	@go install -ldflags='$(LDFLAGS)' ./cmd/...
 
+create-release-artifacts:
+	for os in darwin linux windows; do \
+		zqdir=zq-$(VERSION).$${os}-amd64 ; \
+		rm -rf dist/$${zqdir} ; \
+		mkdir -p dist/$${zqdir} ; \
+		GOOS=$${os} GOARCH=amd64 go build -ldflags='$(LDFLAGS)' -o dist/$${zqdir} ./cmd/... ; \
+	done
+	rm -rf dist/release && mkdir -p dist/release
+	cd dist && for d in zq-$(VERSION)* ; do \
+		zip -r release/$${d}.zip $${d} ; \
+	done
+
+publish-release-artifacts:
```

Comment:
If leaving as is, I suggest you add create-release-artifacts and publish-release-artifacts to the .PHONY list below.

Reply:
In the update to use github assets, I did update the phony line.